crates/db_schema/src/utils.rs

```diff
@@ -51,7 +51,7 @@ use url::Url;
 const FETCH_LIMIT_DEFAULT: i64 = 10;
 pub const FETCH_LIMIT_MAX: i64 = 50;
 pub const SITEMAP_LIMIT: i64 = 50000;
-pub const SITEMAP_DAYS: i64 = 31;
+pub const SITEMAP_DAYS: Option<TimeDelta> = TimeDelta::try_days(31);
```

Comment:
Move the expect here so we dont have to repeat it in other places using the const.

Reply:
I can't unfortunately, rust considers that an unstable feature.